app/.env.example.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Just has an example of a https://github.com/cachethq/Cachet/blob/master/INSTALL.md kind of what dotenv does but just to reference not any use.